client/components/CurrentWord.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import gameActions from '../store/actions/game';
+
+const useStyles = makeStyles((theme) => ({
+  word: {
+    fontSize: 14,
+    textAlign: 'center',
+    wordBreak: 'break-all',
+    [theme.breakpoints.up('sm')]: {
```

Comment:
the touch targets might be a little small on mobile. It's also not very discoverable, currently. Both could be good follow-up CLs.

Reply:
True, maybe a different approach would be better on mobile? Maybe open a dialog to select the character to reveal? That would give us more space.. Definitely a follow-up :-)